_dashboards/workspace/workspace-acl.md

```diff
@@ -151,3 +151,47 @@ To remove several collaborators simultaneously, follow these steps:
 2. Select the **Delete x collaborators** button.
 3. Review the confirmation modal that appears.
 4. Select **Confirm** to remove all selected collaborators from the table.
+
+## Configuring workspace privacy
+
+When permission control is enabled, workspace administrators can set who has access to a workspace, with the following three access levels:
+
+* **Private to collaborators (Default):** Gives only workspace collaborators can access the workspace. 
+* **Anyone can view:** Grants **Read only** permissions to all users in the workspace, allowing them to view workspace assets. 
+* **Anyone can edit:** Grants **Read and write** permissions to all users, allowing view, create, and update workspace assets.
+
+Collaborators are granted higher permissions when their individual access level differs from that set in the workplace settings. For example, if workspace privacy is set to “Anyone can edit”, any collaborator with read-only access level will also be able to edit the workspace assets.
+
+You can set up workspace privacy on the **Create Workspace** page as a **Dashboard admin**. You can also modify it on the **Collaborators** and **Workspace details** pages, as a **Workspace admin** or **Dashboard admin**.
+
+### Setting up workspace privacy during workspace creation
+
+Use the following steps to change workplace privacy settings when creating a new workspace:
+
+1. Choose the desired access level from the cards at the **Set up privacy** panel. 
+2. _Optional_ Decide whether to continue configuring collaborators after workspace creation by selecting the **Go to configure the collaborators right after creating the workspace.** checkbox.
```

Comment:
What do we mean by "configuring collaborators" here? The sentence is unclear. Additionally, we should use shorter and clearer text for the checkbox.

Reply:
@Kapian1234: Where would be the best place to leave feedback on the UI text?

Reply:
@Naarcha-AWS @Kapian1234 It should be something like "Add collaborators after workspace creation"